source/fundamentals/bson/utf8-validation.txt

```diff
@@ -25,15 +25,16 @@ processing overhead since it needs to check the data.
 If you *disable* validation, your application avoids the validation processing
 overhead, but cannot guarantee consistent presentation of invalid UTF-8 data.
 
-The driver enables UTF-8 validation by default. It checks documents for any
-characters that are not encoded in a valid UTF-8 format when it transfers data
-between your application and MongoDB.
+By default, the driver enables UTF-8 validation on data from MongoDB. 
+It checks incoming documents for any characters that are not encoded in a 
+valid UTF-8 format when it parses data sent from MongoDB to your application.
 
 .. note::
 
-   The current version of the {+driver-short+} automatically substitutes
-   invalid UTF-8 characters with alternate valid UTF-8 ones before
-   validation when you send data to MongoDB. Therefore, the validation
+   The current version of the {+driver-short+} automatically substitutes invalid
```

Comment:
S: Since we use the term "current" to refer to the currently active driver version, using it here could cause confusion once later versions are released. I think we can reword it so that we don't need to change it again when this version is a bit older
```suggestion
   This version of the {+driver-short+} automatically substitutes invalid
```